ansible/group_vars/nimbus.kiln.yml

```diff
@@ -53,6 +53,9 @@ beacon_node_discovery_port: '{{ 9000 + idx|int + 1 }}'
 beacon_node_listening_port: '{{ 9000 + idx|int + 1 }}'
 beacon_node_metrics_port:   '{{ 9200 + idx|int + 1 }}'
 beacon_node_rest_port:      '{{ 9300 + idx|int + 1 }}'
+# Light client server
+beacon_node_serve_light_client_data: true
+beacon_node_import_light_client_data: 'only-new'
```

Comment:
Remember to update the variable names here. Also branch name once fix is merged.